src/pages/legal/privacy.jsx

```diff
@@ -0,0 +1,151 @@
+var React = require('react/addons');
+var Footer = require('../../components/footer/footer.jsx');
+
+module.exports = React.createClass({
+  mixins: [
+    React.addons.LinkedStateMixin
+  ],
+  render: function () {
+    return (
+      <div id="splash">
+        <div id="masthead">
+          <div className="inner">
+            <div id="header">
+              <a href="#/"><img src="./img/logo.png" width="200" height="53" alt="Webmaker" /></a>
+            </div>
+          </div>
+        </div>
+
+        <div id="mid">
+          <div className="inner legal">
+            <h1>Webmaker Privacy</h1>
+            <h3>Updated on: June 15, 2013</h3>
+            <p>We care about your privacy. When Mozilla (that’s us) collects information
+              about you from Webmaker, our privacy policy describes how we handle that information.</p>
+
+            <h2>Things you should know</h2>
+
+            <ul>
+              <li>This site and the ones linked to it or embedded in it uses web
+                technologies that may store data in your browser, including “cookies”,
+                which are small pieces of data stored in your browser.
+                <ul>
+                  <li>Webmaker uses cookies to provide its functionality
+                    (such as remembering you between sessions) and to help us understand
+                    how users use the service through aggregate analytics.</li>
+                  <li>Other users may embed content from other online services which
+                    may include cookies and other web technologies. You should check the
+                    privacy policies of those services to understand their privacy practices.</li>
+                </ul>
+              </li>
+              <li>You may elect to receive email from us and can unsubscribe from any
+                email marketing communications you elected to receive.</li>
+              <li>You must create an account to publish content. All the content you
+                publish will be accessible publicly. Profile information you choose to
+                place on public portions of Webmaker (like your username) may be viewed
+                by others.</li>
+              <li>Webmaker may allow you to schedule a time and place for events
+                (such as hackathons) or find out where these types of events are
+                occurring. Mozilla does not have any control over events organized
+                by users so please take care if you attend an event.</li>
+            </ul>
+
+            <h2>Mozilla Privacy Policy</h2>
+
+            <p>Your privacy is an important factor that Mozilla (that's us) considers
```

Comment:
weird mixing of plain apostrophe/quotes and curly quotes. The above text uses `that’s us`, this text uses `that's us`... which style do we want to keep?